backend/prisma/migrations/20211008042337_init/migration.sql

```diff
@@ -0,0 +1,102 @@
+-- CreateTable
```

Comment:
Don't push backend/prisma/migrations. This will make other people reset their database when they migrate.
The .gitignore file has this directory, but git still tracks this file until you remove this directory from git cache or something. You can refer this website
https://www.git-tower.com/learn/git/faq/ignore-tracked-files-in-git

Reply:
I think I did this right, but could you double check when you have a second?

Reply:
Sure. I think we can go over it when we have the job feed meeting